backend/workflow_manager/endpoint/constants.py

```diff
@@ -1,7 +1,7 @@
 class TableColumns:
     CREATED_BY = "created_by"
     CREATED_AT = "created_at"
-    PERMANENT_COLUMNS = ["created_by", "created_at"]
+    PERMANENT_COLUMNS = ["created_by", "created_at", "file_path", "execution_id"]
```

Comment:
@kirtimanmishrazipstack we agreed that we'll allow these column names to be modifiable by the user similar to the `data` column. Can you make that change?

Reply:
@chandrasekharan-zipstack done